presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/PhoenixQueryRunner.java

```diff
@@ -0,0 +1,148 @@
+
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put("connection-url", server.getJdbcUrl())
+                .build();
+
+        queryRunner.installPlugin(new PhoenixPlugin());
+        queryRunner.createCatalog("phoenix", "phoenix", properties);
+
+        if (!tpchLoaded) {
+            createSchema(server, TPCH_SCHEMA);
+            copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, createSession(), tables);
+            tpchLoaded = true;
+        }
+
+        return queryRunner;
+    }
+
+    public static void createSchema(TestingPhoenixServer phoenixServer, String schema)
+            throws SQLException
+    {
+        try (Connection connection = DriverManager.getConnection(phoenixServer.getJdbcUrl());
+                Statement statement = connection.createStatement()) {
+            statement.execute(format("CREATE SCHEMA %s", schema));
+        }
+        catch (Exception e) {
+            throw new IllegalStateException("Can't create schema: " + schema, e);
+        }
+    }
+
+    private static void copyTpchTables(
+            QueryRunner queryRunner,
+            String sourceCatalog,
+            String sourceSchema,
+            Session session,
+            Iterable<TpchTable<?>> tables)
+    {
+        LOG.debug("Loading data from %s.%s...", sourceCatalog, sourceSchema);
+        for (TpchTable<?> table : tables) {
+            copyTable(queryRunner, sourceCatalog, session, sourceSchema, table);
+        }
+    }
+
+    private static void copyTable(
+            QueryRunner queryRunner,
+            String catalog,
+            Session session,
+            String schema,
+            TpchTable<?> table)
+    {
+        QualifiedObjectName source = new QualifiedObjectName(catalog, schema, table.getTableName());
+        String target = table.getTableName();
+
+        @Language("SQL")
+        String sql;
+        switch (target) {
+            case "customer":
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+            case "lineitem":
+                sql = format("CREATE TABLE %s WITH (ROWKEYS = 'ORDERKEY,LINENUMBER', SALT_BUCKETS=10) AS SELECT * FROM %s", target, source);
+                break;
+            case "orders":
+                sql = format("CREATE TABLE %s WITH (SALT_BUCKETS=10) AS SELECT * FROM %s", target, source);
+                break;
+            case "part":
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+            case "partsupp":
+                sql = format("CREATE TABLE %s WITH (ROWKEYS = 'PARTKEY,SUPPKEY') AS SELECT * FROM %s", target, source);
+                break;
+            case "supplier":
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
+                break;
+            default:
+                sql = format("CREATE TABLE %s AS SELECT * FROM %s", target, source);
```

Comment:
You could use the `switch` to define `String tableProperties` (either `""` or `"WITH (ROWKEYS ...)"`)
This way, the code would be shorter and your intent clearer